README.md

```
@@ -217,18 +217,85 @@ to work: when the server is down, Oro simply stops rendering the client-side syn
+```bash
+cd my-project
+orobox create bundle 'Acme\Bundle\FooBundle'
+```
+The argument is the bundle **namespace**, and where the bundle lands is read from the
+`composer.json` in the current directory. An OroCommerce application autoloads
+`"": "src/"`, so inside a project checkout the command above writes
+`src/Acme/Bundle/FooBundle/`. The project already autoloads that path and Oro's kernel
+discovers the generated `Resources/config/oro/bundles.yml`, so the bundle needs no
+`composer.json` of its own — clear the cache and it is loaded:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)